net-misc/scponly/files/scponly-4.8-C23.patch

```diff
@@ -0,0 +1,26 @@
+diff -ru a/helper.c b/helper.c
+--- a/helper.c	2025-01-10 17:16:03.332900358 +0400
++++ b/helper.c	2025-01-10 17:17:24.801652374 +0400
+@@ -1,7 +1,9 @@
+ /*
+  *	helper functions for scponly
+  */
++#define _DEFAULT_SOURCE /* for strsep */
```

Comment:
_DEFAULT_SOURCE is meh because it's a (convoluted) glibc state-machiney-thing, see https://www.gnu.org/software/libc/manual/html_node/Feature-Test-Macros.html (especially the bottom). `_GNU_SOURCE` is usually the best bet if upstream aren't picky.

Reply:
It's a horrible remain of -std=c23 instead of -std=gnu23, due to failure to read.